modules/rosa-autoscaling-nodes.adoc

```diff
@@ -0,0 +1,61 @@
+:_module-type: PROCEDURE
+//Specify the _module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// rosa-nodes/rosa-autoscale-nodes.adoc
+
+[id="rosa-autoscaling_{context}"]
+= Autoscaling nodes in a cluster
+
+[role="_abstract"]
+Configure autoscaling to increase the number of worker nodes that are available in a cluster. Autoscaling for worker nodes is configured in the machine pool definition.
+
+[NOTE]
+====
+Autoscaling is dependent on the resource quotas that are set. If there are insufficient resources, worker nodes are removed from cluster autoscaling when pods fail. Verify resource quotas and request quota increases from the link:https://aws.amazon.com/console/[AWS console].
+====
+
+.Procedure
+
+. To identify the machine pool IDs in a cluster, enter the following command:
++
+[source,terminal]
+----
+$ rosa list machinepools --cluster=<cluster_name>
+----
++
+.Example output
+[source,terminal]
+----
+ID        AUTOSCALING   REPLICAS    INSTANCE TYPE  LABELS    TINTS   AVAILABILITY ZONES
+mp1       No            2           m5.xlarge                        us-east-1a
+----
++
+Use the output to determine which machine pool ID to configure for autoscaling.
+
+. To enable autoscaling on a machine pool, using the default minimum and maximum of 2 replicas, enter the following command:
++
+[source,terminal]
+----
+$ rosa edit machinepool --name=<machinepool_ID> --enable-autoscaling
+----
++
+Alternatively, set the minimum and maximum number of replicas to a non-default setting by entering the following command:
++
+[source,terminal]
+----
+$ rosa edit machinepool --cluster=<cluster_name> --name=<machinepool_ID> --enable-autoscaling --min-replicas=<number> --max-replicas=<number>
+----
++
+.Example
+Enable autoscaling on a machine pool with the ID `mp1` on a cluster named `mycluster`, with the number of replicas set to scale between 2 and 5 worker nodes:
+[source,terminal]
+----
+$ rosa edit machinepool --cluster=mycluster --name=mp1 --enable-autoscaling --min-replicas=2 --max-replicas=5
+----
++
+[NOTE]
```

Comment:
Right now it looks like this NOTE is trying to be kept with step 2, but it sounds like information that is more applicable to the overall task.

I'd consider moving the note to the top, or at the very least, un-pairing it with step 2 here. If you do move the note to the top, you could consider whether the note that is up there already really needs to be in a note, or if it could just be normal text (just so that there aren't back-to-back note blocks).

Reply:
Great idea. Done.